src/EventStore.Core/ClusterVNodeStartup.cs

```diff
@@ -154,7 +155,12 @@ IServiceProvider IStartup.ConfigureServices(IServiceCollection services) => Conf
 						.AddGrpc()
 						.AddServiceOptions<Streams>(options =>
 							options.MaxReceiveMessageSize = TFConsts.EffectiveMaxLogRecordSize)
-						.Services,
+						.Services.Configure<KestrelServerOptions>(options => {
+							options.Limits.MaxConcurrentConnections = 5000;
+							options.Limits.MaxConcurrentUpgradedConnections = 5000;
+							options.Limits.Http2.InitialConnectionWindowSize = 131072 * 1024;
+							options.Limits.Http2.InitialStreamWindowSize = 98304 * 1024;
+						}),
```

Comment:
These need to be moved to the hostbuilder to have an effect

Reply:
They should really have configuration options too

Reply:
not sure they should have cong. options

Reply:
I've moved the settings and added configuration options in the same place as the keepAlive settings